bench-tps/src/main.rs

```diff
@@ -146,7 +146,9 @@ fn main() {
         target /= MAX_SPENDS_PER_TX;
     }
     let gen_keypairs = rnd.gen_n_keypairs(total_keys as u64);
-    let barrier_id = rnd.gen_n_keypairs(1).pop().unwrap();
+    let mut barrier_ids = rnd.gen_n_keypairs(2);
+    let barrier_source_id = barrier_ids.pop().unwrap();
```

Comment:
Seems like you could do this in 2 lines instead of 3 by using Keypair::new()...?